Tests/LiquidTests/XCTestCase+Extensions.swift

```diff
@@ -8,8 +8,11 @@
 import XCTest
 @testable import Liquid
 
-func XCTAssertTemplate(_ templateString: String, _ expression2: String, _ values: [String: Any?] = [:], fileSystem: FileSystem = BlankFileSystem(), _ message: @autoclosure () -> String = "", file: StaticString = #file, line: UInt = #line) {
+func XCTAssertTemplate(_ templateString: String, _ expression2: String, _ values: [String: Any?] = [:], fileSystem: FileSystem = BlankFileSystem(), filters: [String: FilterFunc]? = nil, _ message: @autoclosure () -> String = "", file: StaticString = #file, line: UInt = #line) {
 	let template = Template(source: templateString, fileSystem: fileSystem)
+	if let filters = filters {
```

Comment:
`filters?.forEach { ... }` should work too, skipping the if-let
or making filters non optional with a default value of `[:]`